configure.ac

```diff
@@ -20,7 +20,7 @@ ERFA_NUMVER
 ERFA_LIB_VERSION_INFO(8, 0, 7)
 
 ## SOFA version, update if needed in new relases
-AC_DEFINE([SOFA_VERSION], ["20190722"], [Define to the version of SOFA])
+AC_DEFINE([SOFA_VERSION], ["20200721"], [Define to the version of SOFA])
```

Comment:
I think you should also bump the version number (few lines above) to "1.8.0".
Is it correct?

Reply:
Yes, absolutely right, but supposedly this is done in the next step, of actually preparing a release... (I'm trying to follow `RELEASE.txt`)

Reply:
ups, sorry.
I didn't read `RELEASE.txt` before writing.